### PR TITLE
Improve Docker and Kubernetes support for KNX

### PR DESCRIPTION
**Pull request by plomosits**
## Proposed change

Document new config parameter for pull request https://github.com/home-assistant/core/pull/48065

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/48065
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/knx.markdown [hunk 1]
```diff
@@ -146,6 +146,11 @@ local_ip:
   description: IP address of the local interface.
   type: string
   required: false
+route_back:
+  description: When True the KNXnet/IP Server shall use the IP address and the port number from the IP package received as the target IP address or port number for the response to the KNXnet/IP Client (for NAT / Docker).
+  type: boolean
+  default: true
```
Comment (farmio) [inline source/_integrations/knx.markdown]: default is `false`

**Review by farmio — CHANGES_REQUESTED** [review]
This is targeted against the wrong branch. `next` would be right imho.

File: source/_integrations/knx.markdown (hunk 1 above)
Comment (plomosits) [reply source/_integrations/knx.markdown]: You are right. Thank you.

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelated commits in this PR, due to the change of target branch. Please git rebase.

**Comment by frenck** [comment]
Seems like you performed a merge instead of a rebase, as there are now more unrelated commits. Please make sure you perform a rebase.

**Comment by farmio** [comment]
Hey! 
When you're stuck in a rebase/merge dilemma here (which happens to me more often than I'd like to admit) you could probably just do
```shell
# update upstream
git fetch upstream
# discard all changes, clear the branch, set it to the target
git reset --hard upstream/next
# apply your first commit
git cherry-pick e077e98
# apply the second too
git cherry-pick a963d75
# push it
git push --force
```
good luck!

**Review by farmio — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @plomosits 👍

**Pull request merged**